src/include/ipc4/base_fw.h

```diff
@@ -485,7 +485,7 @@ struct ipc4_astate_table {
 struct ipc4_dma_control {
 	uint32_t node_id;
 	uint32_t config_length;
-	uint32_t config_data[1];
+	uint32_t config_data[0];
```

Comment:
Ok, so this is really variable size - can we add doxygen to this structure to better explain usage.
@ujfalusi any impact on kernel ?